src/github.com/getlantern/bytecounting/bytecounting_conn.go

```diff
@@ -0,0 +1,58 @@
+// package bytecounting provides mechanisms for counting bytes read/written on
+// net.Conn and net.Listener.
```

Comment:
No, the package also provides an implementation of net.Listener that wraps the connections from a standard net.Listener with the byte counting functionality.

https://github.com/getlantern/flashlight-build/blob/tcpproxying/src/github.com/getlantern/bytecounting/bytecounting_listener.go